tests/pr_test.go

```diff
@@ -25,6 +25,10 @@ const yamlLocation = "../common-dev-assets/common-go-assets/common-permanent-res
 
 var permanentResources map[string]interface{}
 
+var ignoreUpdates = []string{
+	"module.postgresql_db.ibm_database.postgresql_db",
+}
+
```

Comment:
We can't really ignore this, as this is the postgrs module. I guess you ignored it because upgrade test failed with this:
```
      	Messages:   	Resource(s) identified to be updated Name: postgresql_db Address: module.postgresql_db.ibm_database.postgresql_db Actions: [update]
        	            	DIFF:
        	            	Before: {"configuration":null}
        	            	After: {"configuration":"{\"max_connections\":250}"}
```

This is expected because we changed the example, so I'd rather we just skipped the upgrade test for this PR only instead of merging code to ignore it

Reply:
you are right, I removed the code and added skip into commit message. Thanks